src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -441,6 +442,10 @@ bool runtime::intelgpu::IntelGPUBackend::compile(shared_ptr<Function> func)
             function_output_names.push_back(get_input_name(op));
             break;
         }
+        case OP_TYPEID::EmbeddingLookup:
```

Comment:
Please move this line to the end of the `switch` into line 1515.

Reply:
done